frontend/src/agenda/agenda.vue

```diff
@@ -88,38 +216,48 @@
           this.getBookings()
         })
       },
-      viewRender(view, el) {
+      viewRender(view, el, cc) {
```

Comment:
cc is unused here?

Reply:
yes... and viewRender always receives exactly two parameters... will remove.